v2/internal/controllers/edge_case_test.go

```diff
@@ -225,24 +225,64 @@ func Test_Owner_IsImmutableOnceSuccessfullyCreated(t *testing.T) {
 	tc.DeleteResourceAndWait(acct)
 }
 
-func Test_AzureName_IsMutableIfNotSuccessfullyCreated(t *testing.T) {
+//func Test_AzureName_IsMutableIfNotSuccessfullyCreated(t *testing.T) {
+//	t.Parallel()
+//
+//	tc := globalTestContext.ForTest(t)
+//	rg := tc.CreateTestResourceGroupAndWait()
+//
+//	invalidAzureName := "==--039+"
+//
+//	acct := createStorageAccount(tc, rg)
+//	acct.Spec.AzureName = invalidAzureName
+//	tc.CreateResourceAndWaitForFailure(acct)
+//
+//	// Patch the account to change AzureName
+//	old := acct.DeepCopy()
+//	acct.Spec.AzureName = tc.NoSpaceNamer.GenerateName("stor")
+//	tc.PatchResourceAndWait(old, acct)
+//
+//	tc.Expect(acct.Owner().Name).ToNot(BeIdenticalTo(invalidAzureName))
+//	// Delete the account
+//	tc.DeleteResourceAndWait(acct)
+//}
```

Comment:
Should either uncomment or delete this test.

Reply:
Fixed